sys/linux/dev_vga_arbiter_386.const

```diff
@@ -0,0 +1,4 @@
+# AUTOGENERATED FILE
+# AT_FDCWD is not set
```

Comment:
Are we missing the include for `AT_FDCWD`?

Reply:
Thanks for the catch. Not sure why it pass the test though.